### PR TITLE
Adding dockershim to glossary.

### PR DESCRIPTION
**Pull request by miteshskj**
Fixes: https://github.com/kubernetes/website/issues/32936

### CODE REVIEW

**Comment by tengqm** [comment]
Nice to have this.
/lgtm

File: content/en/docs/reference/glossary/dockershim.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+Dockershim is a component of Kubernetes version 1.23 and earlier. It allowed Kubernetes system components like kubelet to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
+
+<!--more-->
+
+Starting with version 1.24, dockershim has been removed from Kubernetes. For more information, see [Dockershim FAQ](/blog/2022/02/17/dockershim-faq).
```
Comment (chrisnegus) [inline content/en/docs/reference/glossary/dockershim.md:17]:
```suggestion
Starting with version 1.24, dockershim has been removed from Kubernetes. For more information, see [Dockershim FAQ](/dockershim).
```

**Review by chrisnegus — COMMENTED** [review]
Small suggestion. Otherwise LGTM.

File: content/en/docs/reference/glossary/dockershim.md (hunk 1 above)
Comment (miteshskj) [reply content/en/docs/reference/glossary/dockershim.md]: Thanks @chrisnegus . Updated the url.

File: content/en/docs/reference/glossary/dockershim.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Dockershim
+id: dockershim
+date: 2022-04-15
+full_link: /blog/2022/02/17/dockershim-faq/
+short_description: >
+  Dockershim is a component of Kubernetes v1.23 and earlier, which allows Kubernetes system components to communicate with Docker Engine.
```
Comment (sftim) [inline content/en/docs/reference/glossary/dockershim.md:7]:
```suggestion
   A component of Kubernetes v1.23 and earlier, which allows Kubernetes system components to communicate with Docker Engine.
```

File: content/en/docs/reference/glossary/dockershim.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+aka:
+tags:
+- fundamental
+---
+Dockershim is a component of Kubernetes version 1.23 and earlier. It allowed Kubernetes system components like kubelet to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
```
Comment (sftim) [inline content/en/docs/reference/glossary/dockershim.md:13]: As this is targeting the _main_ branch, we should write something like:
```suggestion
The dockershim is a component of Kubernetes version 1.23 and earlier. It allows the kubelet
to communicate with {{< glossary_tooltip text="Docker Engine" term_id="docker" >}}.
```

and then file an issue to update this text for the v1.24 release. We should also backport that change to the v1.23 docs after the v1.24 release has happened.

File: content/en/docs/reference/glossary/dockershim.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Dockershim
+id: dockershim
+date: 2022-04-15
+full_link: /blog/2022/02/17/dockershim-faq/
```
Comment (sftim) [inline content/en/docs/reference/glossary/dockershim.md:5]:
```suggestion
Either omit the `full_link`, or change it to `/dockershim`.
```

**Review by sftim — COMMENTED** [review]
I recommend a few more changes.

**Review by sftim — COMMENTED** [review]
/lgtm

Thanks.

**Pull request merged**